c/contracts/meta_contract.c

```diff
@@ -109,7 +109,10 @@ int main() {
     _gw_fast_memcpy((uint8_t *)(&fee_amount), (uint8_t *)amount_seg.ptr,
                     sizeof(uint128_t));
 
-    uint32_t reg_id = *(uint32_t *)reg_id_seg.ptr;
```

Comment:
In `c/contracts` dir, there are some other places used this kind of scheme.


```C
c/contracts/eth_addr_reg.c:
  108      mol_seg_t reg_id_seg = MolReader_Fee_get_registry_id(&fee_seg);
  109:     uint32_t reg_id = *(uint32_t *)reg_id_seg.ptr;
  110  

  141      mol_seg_t reg_id_seg = MolReader_Fee_get_registry_id(&fee_seg);
  142:     uint32_t reg_id = *(uint32_t *)reg_id_seg.ptr;
  143  

c/contracts/meta_contract.c:
   81  
   82:     uint32_t reg_id = *(uint32_t *)reg_id_seg.ptr;
   83      ret = handle_fee(&ctx, reg_id, fee_amount);

  111  
  112:     uint32_t reg_id = *(uint32_t *)reg_id_seg.ptr;
  113      ret = handle_fee(&ctx, reg_id, fee_amount);

c/contracts/sudt.c:
  91  
  92:     uint32_t reg_id = *(uint32_t *)fee_reg_seg.ptr;
  93      uint32_t from_id = ctx.transaction_context.from_id;
```

Reply:
We can fix these in different pr.